tests/scenarios/conftest.py

```
@@ -0,0 +1,54 @@
+import pytest
+from faker import Faker
+from pytest_bdd import given
+from selenium import webdriver
+from constants.constant import Constant
+from driver.config import Config
+from msedge.selenium_tools import Edge, EdgeOptions
+
+
+@pytest.fixture
+def driver():
+    """This method is used to open and close the driver"""
+    try:
+        if Config.DRIVER == "chrome":
+            chromeOptions = webdriver.ChromeOptions()
+            chromeOptions.add_argument('--headless')
+            driver = webdriver.Chrome(Config.CHROME_DRIVER_PATH, chrome_options=chromeOptions)
+        elif Config.DRIVER == "firefox":
+            fireFoxOptions = webdriver.FirefoxOptions()
+            fireFoxOptions.add_argument('--headless')
+            driver = webdriver.Firefox(executable_path=Config.FIREFOX_DRIVER_PATH, firefox_options=fireFoxOptions)
+        elif Config.DRIVER == "msedge":
+            edge_options = EdgeOptions()
+            edge_options.use_chromium = True
+            edge_options.add_argument('--headless')
+            driver = Edge(executable_path=Config.MS_EDGE_DRIVER_PATH, options=edge_options)
+        else:
+            raise RuntimeError
```

Try to raise an exception for this scenario
 If the user provides any different browser name we need to raise an exception like The given browser is not supported. Browser name must be parametrized.

---

Updated